subsys/bluetooth/services/ots/ots_l2cap.c

```diff
@@ -90,6 +90,21 @@ static void l2cap_sent(struct bt_l2cap_chan *chan)
 	}
 }
 
+int l2cap_recv(struct bt_l2cap_chan *chan, struct net_buf *buf)
```

Comment:
Just a heads up about: https://github.com/zephyrproject-rtos/zephyr/pull/35958

Let me know if there is something that you would like to see changed in that PR.

Reply:
I think it looks fine as is, depending on which PR goes in first, there may be a conflict to resolve but I think without the actual receive functionality built into OTS the change you have makes sense.